arbeitszeit/use_cases/get_statistics.py

```diff
@@ -24,15 +34,48 @@ class GetStatistics:
     company_repository: CompanyRepository
     member_repository: MemberRepository
     plan_repository: PlanRepository
+    cooperation_respository: CooperationRepository
+    account_respository: AccountRepository
 
     def __call__(self) -> StatisticsResponse:
+        (
+            certs_total,
+            available_product,
+        ) = self._count_certificates_and_available_product()
         return StatisticsResponse(
             registered_companies_count=self.company_repository.count_registered_companies(),
             registered_members_count=self.member_repository.count_registered_members(),
+            cooperations_count=self.cooperation_respository.count_cooperations(),
+            certificates_count=certs_total,
+            available_product=available_product,
             active_plans_count=self.plan_repository.count_active_plans(),
             active_plans_public_count=self.plan_repository.count_active_public_plans(),
             avg_timeframe=self.plan_repository.avg_timeframe_of_active_plans(),
             planned_work=self.plan_repository.sum_of_active_planned_work(),
             planned_resources=self.plan_repository.sum_of_active_planned_resources(),
             planned_means=self.plan_repository.sum_of_active_planned_means(),
         )
+
+    def _count_certificates_and_available_product(self) -> Tuple[Decimal, Decimal]:
+        """
+        available product is sum of prd account balances *(-1)
+        available certificates is sum of company work account balances and sum of member account balances
+        """
+        certs_in_company_accounts = Decimal(0)
+        available_product = Decimal(0)
+        all_companies = self.company_repository.get_all_companies()
```

Comment:
This code would be a little bit cleaner if two methods were factored out: `_count_certs_and_products_from_companies() -> Tuple[int,int]` and `_count_certs_in_member_accounts() -> int`